lib/apiservers/engine/backends/container.go

```diff
@@ -1336,7 +1336,36 @@ func (c *Container) ContainerLogs(ctx context.Context, name string, config *back
 // ContainerStats writes information about the container to the stream
 // given in the config object.
 func (c *Container) ContainerStats(ctx context.Context, name string, config *backend.ContainerStatsConfig) error {
-	return fmt.Errorf("%s does not yet implement ContainerStats", ProductName())
+	defer trace.End(trace.Begin(name))
+
+	// Look up the container name in the metadata cache to get long ID
+	vc := cache.ContainerCache().GetContainer(name)
+	if vc == nil {
+		return NotFoundError(name)
+	}
+
+	// get the configured CPUMhz for this VCH so that we can calculate docker CPU stats
+	cpuMhz, err := systemBackend.SystemCPUMhzLimit()
```

Comment:
I don't recall what we were wanting to report here - was it the percentage consumed of the entire VCH (resource pool) limit or the percentage of the limit imposed on that containerVM (nominal vCPU MHz * number vCPU / ...)